tests/test_core.py

```diff
@@ -0,0 +1,69 @@
+from gendiff.core import generate_diff
+from tests.fixtures.expected_tests_results import EXPECTED_DIFF, EXPECTED_DIFF_NESTED
+from tests.paths import resolve_path
+
+
+def test_generate_diff_json() -> None:
+    # given
+    file1 = str(resolve_path("fixtures/file1.json"))
+    file2 = str(resolve_path("fixtures/file2.json"))
+    expected_diff = EXPECTED_DIFF
```

Comment:
usually expected result is used only in then block (as it's not a part of test input)